random.py

```diff
@@ -250,6 +250,38 @@ def generate_random_date(self):
     def run(self, view, **kwargs):
         self.insert(view, self.generate_random_date)
 
+class RandomTimeCommand(RandomText):
+
+    def generate_random_time(self):
+        hour = r.randint(0, 23)
+        minute = r.randint(0, 59)
+        second = r.randint(0, 59)
+        time = datetime.time(hour, minute, second)
+        return time.isoformat()
+
+    def run(self, view, **kwargs):
+        self.insert(view, self.generate_random_time)
+
+class RandomDatetimeRfc3339Command(RandomText):
+
+    def generate_random_datetime_rfc3339(self):
+        max_year = get_settings().get('max_year', (datetime.datetime.now().year,))[0]
+        min_year = get_settings().get('min_year', (2010,))[0]
+
+        year = r.randint(min_year, max_year)
+        month = r.randint(1, 12)
+        day = r.randint(1, 28)
+        date = datetime.date(year, month, day)
+        hour = r.randint(0, 23)
+        minute = r.randint(0, 59)
+        second = r.randint(0, 59)
+        time = datetime.time(hour, minute, second)
+        timezone = r.randint(0, 24) - 12
```

Comment:
As far as I understand, RFC3339 timezones should be formatted like this: `+05:00Z`

Reply:
Thank you very much, I checked https://www.ietf.org/rfc/rfc3339.txt (pages 8,9) and it is  either Z or numeric zone. I return "Z" for UTC timezone and numeric zone otherwise. Alternatively I could just discard timezone and always return Z, as I believe it is not essential.